valgrind/docker/Dockerfile

```
@@ -0,0 +1,37 @@
+FROM ubuntu:24.04
+
+RUN apt-get update && apt-get install -y git
+RUN git clone https://github.com/theory/pgenv
+
+WORKDIR /pgenv
+RUN git fetch && git checkout v1.3.6b
```

as per https://stackoverflow.com/questions/8932389/git-shallow-clone-to-specific-tag

```suggestion
RUN git clone --branch v1.3.6b --depth 1 https://github.com/theory/pgenv
```

less code, less data, instantly in the state you want it in